support/scripts/schemas.ts

```diff
@@ -117,9 +117,30 @@ async function main() {
   java.forEach((file, path) => {
     writeToFile(javaDir + path, file.lines);
   });
+
+  const php = await quicktype({
+    inputData,
+    lang: "php",
+    inferUuids: false,
+    inferDateTimes: false,
+    rendererOptions: {
+      "acronym-style": "camel",
+      "with-get": false,
+    },
+  });
+
+  const phpDir = "./languages/php/src/Schemas/";
+  if (!fs.existsSync(phpDir)) {
+    fs.mkdirSync(phpDir);
+  }
+
+  php.lines.splice(1, 0, "namespace Bitwarden\\Sdk\\Schemas;", "use stdClass;", "use Exception;");
+
+  writeToFile("./languages/php/src/Schemas/Schemas.php", php.lines);
```

Comment:
Just a nit... isn't the `schemas` directory normally (and currently) lowercase? Any reason we are capitalizing it?

Reply:
namespaces are case sensitive in PHP. We imported with capitalised name (like most PHP projects to), but the directory were lowercase, which is wrong.
Many big PHP projects just use capitalised name, but i don't think there is any namespace naming convention. I would keep it capitalized as many PHP projects does.

Reply:
Few examples from large projects, all use capitalised names:
https://github.com/laravel/laravel/tree/11.x/app
https://github.com/symfony/symfony/tree/7.2/src/Symfony
https://github.com/composer/composer/tree/main/src (we use this)

Reply:
Perfect, thank you!